app/src/main/java/org/fossasia/openevent/general/order/OrderDetailsViewHolder.kt

```diff
@@ -5,16 +5,17 @@ import android.net.Uri
 import android.provider.CalendarContract
 import android.view.View
 import androidx.recyclerview.widget.RecyclerView
-import kotlinx.android.synthetic.main.item_card_order_details.view.calendar
-import kotlinx.android.synthetic.main.item_card_order_details.view.date
-import kotlinx.android.synthetic.main.item_card_order_details.view.eventDetails
 import kotlinx.android.synthetic.main.item_card_order_details.view.eventName
-import kotlinx.android.synthetic.main.item_card_order_details.view.eventSummary
 import kotlinx.android.synthetic.main.item_card_order_details.view.location
+import kotlinx.android.synthetic.main.item_card_order_details.view.date
+import kotlinx.android.synthetic.main.item_card_order_details.view.eventSummary
+import kotlinx.android.synthetic.main.item_card_order_details.view.organizer
 import kotlinx.android.synthetic.main.item_card_order_details.view.map
+import kotlinx.android.synthetic.main.item_card_order_details.view.eventDetails
+import kotlinx.android.synthetic.main.item_card_order_details.view.organizerLabel
 import kotlinx.android.synthetic.main.item_card_order_details.view.name
+import kotlinx.android.synthetic.main.item_card_order_details.view.calendar
 import kotlinx.android.synthetic.main.item_card_order_details.view.orderIdentifier
-import kotlinx.android.synthetic.main.item_card_order_details.view.organizer
```

Comment:
All the above are unnecessary changes.

Reply:
IDE itself made wildcard import, so I manually did all imports

Reply:
That's okay, but then you can put them in the same way they were earlier to prevent these deletions and additions from showing up.

Reply:
I will keep in mind next time :)